src/main/java/appeng/tile/crafting/TileMolecularAssembler.java

```diff
@@ -188,7 +190,7 @@ private boolean hasMats()
 	@Override
 	public boolean acceptsPlans()
 	{
-		return this.inv.getStackInSlot( 10 ).isEmpty();
+		return ItemHandlerUtil.isEmpty(this.patternInv);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)